pkgs/development/beam-modules/hex/default.nix

```diff
@@ -8,13 +8,13 @@ let
 
   pkg = self: stdenv.mkDerivation rec {
     name = "hex";
-    version = "v0.11.3";
+    version = "v0.14.0";
 
     src = fetchFromGitHub {
         owner = "hexpm";
         repo = "hex";
-        rev = "f5e200ad95f030f0a7ab88a86545dd0dde1ee521";
-        sha256 = "0n4cgmnbmglarydls9pmxznbzp49pv85ncbd4f2lp1fm7qr08xfw";
+        rev = "cd6111d29b67bb9db8e6844bef568e65ca4c32a1";
```

Comment:
I think you can just use `${version}` here which fetches the tag `v0.14.0` which is exactly what you want. Then updating is less of a hassle.